packages/drivers/routerlicious-driver/src/restWrapper.ts

```diff
@@ -131,6 +132,14 @@ export class RouterliciousRestWrapper extends RestWrapper {
 		statusCode: number,
 		canRetry = true,
 	): Promise<IR11sResponse<T>> {
+		// Check whether this request has been made before or if it is a retry.
+		const originalRequestUrl = requestConfig.url;
+		const requestKey = `${requestConfig.method ?? ""}|${requestConfig.url ?? ""}`;
```

Comment:
nit: use `originalRequestUrl` instead of `requestConfig.url` again so that when you take `baseUrl` into account, that's also reflected here